rllib/algorithms/impala/impala.py

```diff
@@ -642,15 +651,16 @@ def setup(self, config: AlgorithmConfig):
             )
         else:
             # Create our local mixin buffer if the num of aggregation workers is 0.
-            self.local_mixin_buffer = MixInMultiAgentReplayBuffer(
-                capacity=(
-                    self.config.replay_buffer_num_slots
-                    if self.config.replay_buffer_num_slots > 0
-                    else 1
-                ),
-                replay_ratio=self.config.replay_ratio,
-                replay_mode=ReplayMode.LOCKSTEP,
-            )
+            if self.config.replay_proportion > 0.0:
+                self.local_mixin_buffer = MixInMultiAgentReplayBuffer(
```

Comment:
Should this one be replaced in the future with the MAERB?

Reply:
Yeah, some mixin version of it. :) 
The APPO/IMPALA I have in a PR for the new stack does not support mixin yet (and also not aggregation workers). But yes, we should find out, whether these things really help. I don't think we have good data points yet for these, even on the old stack.